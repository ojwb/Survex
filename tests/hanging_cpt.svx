; pos=fail
*fix 1 0 0 0
1 2  1 0 0

3 4  1 0 0
3 5  1 0 0
3 6  1 0 0
3 7  1 0 0
4 5  1 0 0
6 7  1 0 0
