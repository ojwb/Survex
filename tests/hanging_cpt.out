
Removing trailing traverses...

Concatenating traverses...

Simplifying network...
cavern: error: Survey not all connected to fixed stations
The following survey stations are not attached to a fixed point:
./hanging_cpt.svx:5: info: 3
